src/main/java/org/opensearch/security/action/tenancy/MultiTenancyConfigurationsRestHandler.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.action.tenancy;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.rest.action.RestToXContentListener;
+
+import static org.opensearch.rest.RestRequest.Method.GET;
+import static org.opensearch.rest.RestRequest.Method.PUT;
+
+public class MultiTenancyConfigurationsRestHandler extends BaseRestHandler {
+
+    public MultiTenancyConfigurationsRestHandler() {
+        super();
+    }
+
+    @Override
+    public String getName() {
+        return "Multi Tenancy actions for Retrieve / Update configurations.";
+    }
+
+    @Override
+    public List<Route> routes() {
+        return ImmutableList.of(
+                new Route(GET, "/_plugins/_security/api/tenancy/multitenancy_configurations"),
```

Comment:
Seems like this is a good opportunity to decouple this API from the security plugin if we want to separate tenancy features in the feature, what do you think of `/_tenancy/config`?

If that is too drastic, what about `/_plugins/_security/api/tenancy/config`?

Reply:
Cool. Going with ```/_plugins/_security/api/tenancy/config```